3.0-RELEASE-NOTES.md

```diff
@@ -5,3 +5,14 @@ always describe the impact on users and instructions for upgrading
 applications from 2.x to 3.0.
 
 See also https://github.com/strongloop/loopback/blob/master/3.0-DEVELOPING.md
+
+
+## API call for remote method has been changed.
+
+Originally, we have SharedClass.prototype.find method for looking up remote
```

Comment:
Please wrape `SharedClass.prototype.find` in backticks, so that it is rendered as a code. Same for the other instance below.